agot-bg-game-server/src/common/ingame-game-state/game-data-structure/createGame.ts

```diff
@@ -181,6 +190,16 @@ export default function createGame(ingame: IngameGameState, housesToCreate: stri
         });
     }
 
+    // Another very strange behavior I cannot explain:
+    // I tried to allow defining maxPowerTokens in a generic way like I do with the overwritten super control power token
+    // But for max power tokens it just doesn't work though I verified selectedGameSetup contains the maxPowerTokens node with
+    // console.log(JSON.stringify(selectedGameSetup, null, 2));
+
+    // const maxPowerTokensPerHouse = new BetterMap(selectedGameSetup.maxPowerTokens != undefined ? Object.entries(selectedGameSetup.maxPowerTokens) : []);
+
+    // So for now the only work around is to hard code it:
+    const maxPowerTokensPerHouse = new BetterMap(entireGame.isFeastForCrows ? [["arryn", 19]] : []);
```

Comment:
@Longwelwind Another weird situation I just cannot explain. It's absolutely not urgent, but if you are interested and have time you maybe have an idea...